types/pixi/grids/GridLayer.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * A CanvasLayer responsible for drawing a square grid
+ */
+declare class GridLayer extends CanvasLayer{
```

Comment:
Seems to be missing a few methods here, mostly borrowed from `this.grid` e.g. `getCenter`